src/fonts/OTF/TeX/makeFF

```diff
@@ -917,7 +917,7 @@ $map{cmbsy10} = {
     0x33 => 0x220B,         # \ni
     0x34 => 0x25B3,         # \bigtriangleup and \triangle
     0x35 => 0x25BD,         # \bigtriangledown
-    0x36 => [0x338,-894,0], # \not (combining)
+    0x36 => 0x338,          # \not (combining)
```

Comment:
Did you want to change this comment?

Reply:
I'm not sure.  I had to use `0x338` b/c the non-combing char code for `\not`, `0x2F` was being used by something else.

Reply:
Oh, there's a noncombining character for `\not`? 0x2F looks like a regular slash, Unicode just calls it "solidus".

This makes me wonder whether this character differs from the regular slash glyph.  If not, we could use that one.  But my guess is that it differs?

Reply:
Unicode calls 0x338 a combining long solidus overlay.  The glyph is the same, it's jus that the perl script was complaining about there being multiple mappings when I had this line as `0x36 => 0x2F`.  For the life of me I couldn't find the other mapping though. :\

Reply:
In a future PR I'd like to change back all of those accents that I changed before to be use the combining unicode point.  Since we're using them as combining characters I think it makes more sense.